Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -83,7 +83,7 @@ public partial class MobiledgeXIntegration : IDisposable
         Location fallbackLocation = new Location(0,0);
         CarrierInfoClass carrierInfoClass = new CarrierInfoClass(); // used for IsRoaming check
         MelMessaging melMessaging;
-        internal PersistentConnection persistentConnection;
+        public PersistentConnection persistentConnection;
```

Comment:
because this is public now, we should really change the name. PersistentConnection means nothing to the developer. Maybe "UnityEdgeEvents"?